src/main/java/net/syscon/elite/api/resource/BookingResource.java

```diff
@@ -397,7 +397,6 @@ PrivilegeSummary getBookingIEPSummary(@ApiParam(value = "The booking id of offen
     @GetMapping("/{bookingId}/mainOffence")
     @ApiOperation(value = "Get Offender main offence detail.", notes = "Offender main offence detail.")
     @ApiResponses(value = {
-            @ApiResponse(code = 200, message = "OK", response = OffenceDetail.class, responseContainer = "List"),
```

Comment:
unnecessary annotations that are prone to error - 200 is generated automatically